modules/ossm-auto-route.adoc

```diff
@@ -56,6 +56,14 @@ If you need specific annotations in the OpenShift Routes created by {ProductShor
 
 By default, the `ServiceMeshControlPlane` resource automatically synchronizes the Gateway resources with OpenShift routes. Disabling the automatic route creation allows you more flexibility to control routes if you have a special case or prefer to control routes manually.
 
+== Skipping automatic route creation for specific cases
+
```

Comment:
@jwendell this looks pretty good to me. I only have small suggestions, such as deleting the word special in the phrase "...you must add the special annotation..."